config/aws_media_upload.js

```diff
@@ -0,0 +1,32 @@
+const aws = require('aws-sdk');
+const multer = require('multer');
+const multerS3 = require('multer-s3');
+var path = require('path');
+
+// AWS 
+aws.config.update({
+  secretAccessKey: 'WNM0q+7MEsTL1FSGeaA2W6XzaaRISUp/nQ32tSff',
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears your code contains a hardcoded AWS Secret Access Key. This practice significantly raises the likelihood of unauthorized access to your AWS services, potentially leading to data leakage or misuse of resources. Possible remediation approaches are:
	 1. Manage secret data using environment variables:
		 a) Identify the secret data and store that data in environment variables on your local and production server.
		 b) Replace the secrets with references to environment variables.
	 2. Use AWS Secrets Manager to store and access secrets:
		 a) Visit Secrets Manager on the AWS Management Console.
		 b) Choose the secret type on the console page and follow the instructions.
		 c) Use the suggested code samples to retrieve the secret.

To learn more about secure management of AWS credentials and best practices, see [AWS account management reference guide](https://docs.aws.amazon.com/general/latest/gr/aws-access-keys-best-practices.html).